libs/coin-framework/src/operation.ts

```diff
@@ -225,3 +225,20 @@ export const isAddressPoisoningOperation = (
     operation.value.isZero()
   );
 };
+
+export const isEditableOperation = (
```

Comment:
Something we haven't discussed yet: when should we implement functions in the coin framework vs in live-common ?
Is it the role of the coin-framework the say if an operation is editable or not ?

Reply:
I suggest that we can put it in coin framework for this moment for two reasons.
1. The other helpers related to operation such as "isConfirmedOperation" and "isAddressPoisoningOperation" are implemented in coin framework. So we are doing something similar.
2. The methods in live-common can only be used in LLM and LLD but the methods in coin-framework can also be used by coin specific code in the future. e.g. if we have ethereum specific logic or UI code. (i.e. the "coin-evm" package in the future.) this part of evm specific code can use these helpers("isEditableOperation", ...)

Feel free to ask @sprohaszka-ledger for opinion and feel free to add more commit if you think it is necessary. Thanks

Reply:
My thoughts are:
* if the function has a business logic impact, it can be added to `coin-framework`
* if the function has a UX/UI impact, it should not be in `coin-framework`

We all need to discuss about the scope of `coin-framework` :)

Reply:
ok, thanks a lot for the info.
As this method is UX/UI related for this moment. I move it out of coin-framework